### DISCUSSION TITLE
GL_INVALID_OPERATION: Must have element array buffer bound

### DISCUSSION THREAD

**Discussion by e-hosseini** [visgl/deck.gl#6936] [Q&A]
Hi, I'm working on a library that uses the Deck.gl. On another project, I'm using that library to do the visualization. When I use `npm link` functionality to connect both projects on dev time, everything works fine. But when I install the library like a normal NPM Package, the visualization doesn't happen. It loads the map (google) without any problem. But the DeckOverlay doesn't render the layers, and I can see this log in the console several times:
```
[.WebGL-0x20039d2900] GL_INVALID_OPERATION: Must have element array buffer bound.
```
there is no other relevant error or message.
What do you think about this issue?

**Comment by weige258** [comment]
You should Bind indices buffer before draw
this.ibo.Bind()
this.gl.drawElements(this.gl.TRIANGLES,this.indices!.length,this.gl.UNSIGNED_INT,0)

**Comment by ibgreen** [comment]
There is some chance that may be facing some dependency inconsistency related to the recent release of deck.gl v9.1.

You will want to make sure that your project is using the samne version of deck.gl and luma.gl (v9.0.x or  v9.1.x).

Depending on the latest deck.gl patch versions of either release track should ensure this works automatically.
